app/build.gradle

```diff
@@ -17,23 +17,29 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
-    buildTypes {
+    signingConfigs {
         release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
-            signingConfig signingConfigs.debug
+            storeFile file("keystore.jks")
```

Comment:
What's in this file ? And how it is generated ?

Reply:
This is a security certificate of the application required by FIDO2. It will be documented when I'll be done with the development.

Reply:
The `keystore` contains the private key to sign the application. Does it contains sensitive information ? Because the repository is public.

Reply:
Nope, I don't see any sensitive information stored in it (do you have an idea of data?)

Reply:
Did you generated this file by yourself ? (In this case is will be ok to commit it).

Reply:
Yes, I am the one who generated the file.